docs/sdks/client-sdks/javascript/assignments.mdx

```diff
@@ -0,0 +1,411 @@
+}
+
+// JSX component sets itself based on the campaignJson
+<Hero image={campaignJson.heroImage} title={campaignJson.heroTitle} description={campaignJson.heroDescription} />
+```
+
+## Integer and Numeric Assignments
+
+Integer and numeric assignments work the same way but return either an integer or a floating point number. These assignments are useful where you want to use a numeric value to drive business logic such as pricing on an item or a number of items to display in a list.
+
+```js
+import * as EppoSdk from "@eppo/js-client-sdk";
+
+const client = EppoSdk.getInstance();
+const flagKey = "flag-key-123";
+const subjectKey = getUserId() || "user-123";
+const defaultAssignment = 0; 
+
+// example of getting an integer assignment
+const numberOfItems = client.getIntegerAssignment(
+  flagKey,
+  subjectKey,
+  defaultAssignment
+);
+
+// example of getting a numeric assignment
+const price = client.getNumericAssignment(
+  flagKey,
+  subjectKey,
+  defaultAssignment
+);
+
+// use the assignment to drive business logic
+<ItemList items={numberOfItems} />
+
+// use the assignment to drive business logic
+<Item price={price}/>
+
+```
+
+### Assignment Logger Schema
+
+The SDK will invoke the `logAssignment` function with an `assignment` object that contains the following fields:
+
+<ApiOptionRef 
+  name="timestamp"
+  type="string"
+  defaultValue="undefined"
+>
+
+The time when the subject was assigned to the variation. Example: `"2021-06-22T17:35:12.000Z"`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="featureFlag"
+  type="string"
+  defaultValue="undefined"
+>
+
+An Eppo feature flag key. Example: `"recommendation-algo"`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="allocation"
+  type="string"
+  defaultValue="undefined"
+>
+
+An Eppo allocation key. Example: `"allocation-17"`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="experiment"
+  type="string"
+  defaultValue="undefined"
+>
+
+An Eppo experiment key. Example: `"recommendation-algo-allocation-17"`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="subject"
+  type="string"
+  defaultValue="undefined"
+>
+
+An identifier of the subject or user assigned to the experiment variation. Example: UUID
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="subjectAttributes"
+  type="Record<string, any>"
+  defaultValue="{}"
+>
+
+A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function. Example: `{ "country": "US" }`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="variation"
+  type="string"
+  defaultValue="undefined"
+>
+
+The experiment variation the subject was assigned to. Example: `"control"`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="metaData"
+  type="Record<string, string>"
+  defaultValue="{}"
+>
+
+Metadata around the assignment, such as the version of the SDK. Example: `{ "obfuscated: "true", "sdkLanguage": "javascript", "sdkLibVersion": "3.2.1" }`
+</ApiOptionRef>
+
+### Logging data to your data warehouse
```

Comment:
I think this should be second level (##) and all of the get*Assignment sections above should be level 3 (###)

Right now the right nav looks a bit awkward:

![image](https://github.com/user-attachments/assets/2be9ecf0-87ec-4066-9018-5cda7e55a967)

Reply:
Have changed Assignment Logger Schema to H2.